Lib/test/test_resource.py

```diff
@@ -28,7 +35,8 @@ def test_fsize_ismax(self):
             # the number to a C long long and that the conversion doesn't raise
             # an error.
             self.assertEqual(resource.RLIM_INFINITY, max)
-            resource.setrlimit(resource.RLIMIT_FSIZE, (cur, max))
+            if _support_rlimit_fsize_set:
```

Comment:
Please remove the 2 constants and add a decorator to the function:

@skipIf(sys.platform == "vxworks", "setting RLIMIT_FSIZE is not supported on VxWorks")

And make a similar change for the other function.